octodns/provider/selectel.py

```diff
@@ -0,0 +1,303 @@
+#
+#
+#
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from collections import defaultdict
+
+from logging import getLogger
+
+from requests import Session
+
+from ..record import Record, Update
+from .base import BaseProvider
+
+
+class SelectelAuthenticationRequired(Exception):
+    def __init__(self, msg):
+        message = 'Authorization failed. Invalid or empty token.'
+        super(SelectelAuthenticationRequired, self).__init__(message)
+
+
+class SelectelProvider(BaseProvider):
+    SUPPORTS_GEO = False
+
+    SUPPORTS = set(('A', 'AAAA', 'CNAME', 'MX', 'NS', 'TXT', 'SPF', 'SRV'))
+
+    MIN_TTL = 60
+
+    LIMIT = 50
```

Comment:
A nit, but prefer being a little more explicit here and calling this a page size to make it clear what it applies to.

```suggestion
    PAGINATION_LIMIT = 50
```